services/app-api/src/handlers/apollo_gql.ts

```diff
@@ -365,7 +367,20 @@ const gqlHandler: Handler = async (event, context, completion) => {
     const initializedHandler = await handlerPromise
 
     const response = await initializedHandler(event, context, completion)
-
+    const payloadSize = Buffer.from(event.body).length
+    const serviceName = 'gql-handler'
+    const otelCollectorUrl = process.env.API_APP_OTEL_COLLECTOR_URL
+    if (otelCollectorUrl === undefined || otelCollectorUrl === '') {
+        throw new Error(
+            'Configuration Error: API_APP_OTEL_COLLECTOR_URL is required to run app-api'
+        )
+    }
+    initTracer(serviceName, otelCollectorUrl)
+    if (payloadSize > 5.5 * 1024 * 1024) {
```

Comment:
Sorry, I didn't specify that the response payload is what we need to log and alert for sizes > 5.5mb. I think this block of code is for incoming payload size, which there is a limit, but I'm not sure what it is. 

I think we can keep this alerting for incoming payload size, lets keep it at 5.5mb, and add alerting for outgoing payload. There is a if condition on line 385 that sets up just the console logging, we could place the otel recording there. Please update the size to 5.5mb there too.